serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -136,29 +139,33 @@ For example using VSCode, there are the commands needed:
 
 [source,bash,subs="attributes+"]
 ----
-curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_devmode.yaml > workflow_devmode.yaml
+curl -S https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_sonataflow_devmode.yaml > workflow_devmode.yaml
 code workflow_devmode.yaml
 kubectl apply -f workflow_devmode.yaml -n <your_namespace>
 ----
 
 The operator ensures that the latest Workflow definition is running and ready.
 This way, you can include the Workflow application in your development scenario and start making requests to it.
 
-== Check the Workflow application is running
+== Check if the Workflow is running
 
-In order to check that the {product_name} Greeting application is up and running, you can try to perform a test HTTP call. First, you must expose the service:
+In order to check that the {product_name} Greeting application is up and running, you can try to perform a test HTTP call. First, you must get the service URL:
 
 .Exposing the Workflow application
 [source,bash,subs="attributes+"]
 ----
-kubectl patch svc greeting -n <your_namespace> -p '{"spec": {"type": "NodePort"}}'
 minikube service greeting -n <your_namespace> --url
 http://127.0.0.1:57053
 
 # use the above output to get the current Workflow URL in your environment
 ----
 
-Alter exposing the Workflow service, you can point your browser to the Swagger UI and start making requests with the REST interface.
+[TIP]
+====
+When running on Minikube, the service is already exposed for you via `NodePort`. On OpenShift, link:{openshift_route_url}[a Route is automatically created in devmode]. If you're running on Kubernetes you can link:{kubernetes_url}[expose your service using an Ingress].
+====
+
+You can now point your browser to the Swagger UI and start making requests with the REST interface.
 
 For example, using the above command execution you can access the Swagger UI via `http://127.0.0.1:57053/q/swagger-ui/`.
```

Comment:
I think we could just point people to the chapter around how to test their workflow and remove some of the details around swagger-ui here, as for example the devUI could be used instead as well, or any of the other options.

Reply:
I'll link the other chapter here, but the devUI guide we don't have it yet. 🤔